TrackMyStuffApp/TrackMyStuff.csproj

```diff
@@ -19,6 +19,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Serilog.AspNetCore" Version="3.4.0" />
+    <PackageReference Include="Serilog.Sinks.Graylog" Version="2.2.2" />
```

Comment:
Need to install [Serilog.Sinks.Graylog](https://github.com/whir1/serilog-sinks-graylog) with _nuget package manager_ in order for [GrayLog](https://www.graylog.org/) (CENTRALIZED LOG MANAGEMENT) to work properly.